tests/components/tibber/test_services.py

```diff
@@ -0,0 +1,254 @@
+                                    },
+                                    {
+                                        "startsAt": (
+                                            tomorrow + dt.timedelta(hours=1)
+                                        ).isoformat(),
+                                        "total": 0.46914,
+                                        "level": "VERY_EXPENSIVE",
+                                    },
+                                ],
+                            }
+                        }
+                    }
+                }
+            },
+        ),
+        MagicMock(
+            name="second_home",
+            info={
+                "viewer": {
+                    "home": {
+                        "currentSubscription": {
+                            "priceInfo": {
+                                "today": [
+                                    {
+                                        "startsAt": today.isoformat(),
+                                        "total": 0.46914,
+                                        "level": "VERY_EXPENSIVE",
+                                    },
+                                    {
+                                        "startsAt": (
+                                            today + dt.timedelta(hours=1)
+                                        ).isoformat(),
+                                        "total": 0.46914,
+                                        "level": "VERY_EXPENSIVE",
+                                    },
+                                ],
+                                "tomorrow": [
+                                    {
+                                        "startsAt": tomorrow.isoformat(),
+                                        "total": 0.46914,
+                                        "level": "VERY_EXPENSIVE",
+                                    },
+                                    {
+                                        "startsAt": (
+                                            tomorrow + dt.timedelta(hours=1)
+                                        ).isoformat(),
+                                        "total": 0.46914,
+                                        "level": "VERY_EXPENSIVE",
+                                    },
+                                ],
+                            }
+                        }
+                    }
+                }
+            },
+        ),
+    ]
+    mock_homes[0].name = "first_home"
+    mock_homes[1].name = "second_home"
+    return mock_homes
+
+
+def create_mock_tibber_connection():
+    """Create a mock tibber connection."""
+    tibber_connection = MagicMock()
+    tibber_connection.get_homes.return_value = generate_mock_home_data()
+    return tibber_connection
+
+
+def create_mock_hass():
+    """Create a mock hass object."""
+    mock_hass = MagicMock
+    mock_hass.data = {"tibber": create_mock_tibber_connection()}
+    return mock_hass
```

Comment:
Don't do this. Use our common test helpers and add a mock config entry to hass and patch the library and then set up the config entry.